js/views/messagesitem.js

```diff
@@ -98,6 +98,8 @@ define(function(require) {
 		openMessage: function(event) {
 			event.stopPropagation();
 			$('#mail-message').removeClass('hidden-mobile');
+			// make sure message is marked as read when clicked on it
+			Radio.ui.trigger('messagesview:messageflag:set', this.model.id, 'unseen', false);
```

Comment:
Yeah, I saw that one earlier... 
Maybe it can wait for your huge branch to be merged first...